README.md

```diff
@@ -1,2 +1,18 @@
-# Synergile
- This bot is a project engineered by CodeWritten, with help from Jedi3, Wakfi, and Napkins
+# About us
+ This project is engineered by CodeWritten, Jedi3, Wakfi, and Napkins, (little note from Code here: without you guys, this project wouldn't have taken off, you guys are awesome!)
```

Comment:
```suggestion
 This project is engineered by CodeWritten, Jedi3, Wakfi, and Napkins
```
Appreciate it :3 but this is our public facing description so this is a little out of place; our private dev chat would be a better place to express this